File: Java/_453_Minimum_Moves_to_Equal_Array_Elements/Solution.java
class Solution {
    public int minMoves(int[] nums) {
        int minimum = Integer.MAX_VALUE;
        int res = 0;
        for (int num : nums)
            minimum = Math.min(minimum, num);
        for (int num : nums)
            res += num - minimum;
        return res;
    }
}

File: Python/_453_Minimum_Moves_to_Equal_Array_Elements/solution.py
import math

class Solution:
    def minMoves(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        minimum = math.inf
        res = 0
        for i in range(len(nums)):
            minimum = min(minimum, nums[i])
        for i in range(len(nums)):
            res += nums[i] - minimum
        return res
